src/client/components/ButtonRegisterYourSelf/ButtonRegisterYourSelf.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './ButtonRegisterYourSelf.styles.css';
+
+export const ButtonRegisterYourSelf = (props) => {
```

Comment:
Was this component needed? Couldn't you use the already created `Button` component?